pom.xml

```diff
@@ -177,7 +177,6 @@
       <groupId>org.apache.httpcomponents</groupId>
       <artifactId>httpcore</artifactId>
       <version>4.4</version>
-      <scope>test</scope>
```

Comment:
can you create a test that replicates this bug? or at least send me a stacktrace of the error? thanks